File: ios/RCTMGL-v10/MBXTerrainViewManager.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface MBXTerrain : UIView
{}
@property (nonatomic) NSObject* exaggeration;
@end

@interface RCT_EXTERN_REMAP_MODULE(MBXTerrain, MBXTerrainViewManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(sourceID, NSString);

RCT_EXPORT_VIEW_PROPERTY(reactStyle, NSDictionary);

@end

File: ios/RCTMGL-v10/MBXTerrainViewManager.swift
@objc(MBXTerrainViewManager)
class MBXTerrainViewManager: RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
  
    override func view() -> UIView! {
      let terrain = MBXTerrain()
      terrain.bridge = self.bridge
      return terrain
    }
}
